doc/en/components/grids/_shared/keyboard-navigation.md

```diff
@@ -0,0 +1,337 @@
+ - <kbd>Shift + Tab</kbd> - available only if there is a cell in edit mode; moves the focus to the previous editable cell in the row; after reaching the first cell in the row, moves the focus to the last editable cell in the previous row. When **Row Editing** is enabled, moves the focus from the right-most editable cell to **CANCEL** and **DONE** buttons, and from **DONE** button to the right-most editable cell in the row
+ - <kbd>Space</kbd> - selects the row, if <kbd>Row Selection</kbd> is enabled
+ - <kbd>Alt + Arrow Left</kbd> or <kbd>Alt + Arrow Up</kbd> - 
+ <!-- ComponentStart: Grid --> 
+  over Group Row - collapses the group  
+ <!-- ComponentEnd: Grid --> 
+ <!-- ComponentStart: HierarchicalGrid --> 
+  collapses the row island
+ <!-- ComponentEnd: HierarchicalGrid --> 
+ <!-- ComponentStart: TreeGrid -->
+  collapses the current node
+ <!-- ComponentEnd: TreeGrid -->
+ - <kbd>Alt + Arrow Right</kbd> or <kbd>Alt + Arrow Down</kbd> -  <!-- ComponentStart: Grid --> 
+  over Group Row - expands the group
+ <!-- ComponentEnd: Grid --> 
+ <!-- ComponentStart: HierarchicalGrid --> 
+  expands the row island
+ <!-- ComponentEnd: HierarchicalGrid --> 
+ <!-- ComponentStart: TreeGrid -->
+  expands the current node
+ <!-- ComponentEnd: TreeGrid --> 
+ <!-- ComponentStart: Grid --> 
+ - <kbd>Alt + Arrow Left</kbd> or <kbd>Alt + Arrow Up</kbd> - over Master Detail Row - collapses the details view
+ - <kbd>Alt + Arrow Right</kbd> or <kbd>Alt + Arrow Down</kbd> - over Master Detail Row - expands the details view
+ - <kbd>Space</kbd> - over Group Row - selects all rows in the group, if <kbd>rowSelection</kbd> property is set to multiple
+ <!-- ComponentEnd: Grid --> 
+
+Practice all of the above mentioned actions in the demo sample below. Focus any navigable grid element and a list with some of the available actions for the element will be shown to guide you through.
+
+## Demo
+ <!-- ComponentStart: Grid --> 
+
+<code-view style="height:470px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/grid/grid-keyboardnav-guide" >
+</code-view>
+
+<div class="divider--half"></div>
+ <!-- ComponentEnd: Grid --> 
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<code-view style="height:600px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/hierarchical-grid/hgrid-keyboard-guide" >
+</code-view>
+
+<div class="divider--half"></div>
+<!-- ComponentEnd: HierarchicalGrid -->
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:470px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/tree-grid/tgrid-keyboardnav-guide" >
+</code-view>
+
+<div class="divider--half"></div>
+<!-- ComponentEnd: TreeGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.